### ISSUE TITLE
Support @ sign in context names.

### ISSUE THREAD

**Issue by idcmp** [ahmetb/kubectx#145]
In many environments it's common to have different roles authenticating to the same cluster. This generally results in contexts named "developer@cluster-1" or "admin@cluster-1", etc.

The current completion scripts do not properly digest the '@' sign (treating it as a separator).

**Comment by ahmetb** [comment]
Huh, interesting. Any ideas why @ is treated as a separator?

Also, zsh or bash completion?

**Comment by idcmp** [comment]
I'm using bash. I haven't poked too much into it, I'm guessing '@' is treated as a word separator by default but I don't know where that decision is made.

**Comment by loganmzz** [comment]
Has it been fixed ? I've no issue with Bash and `@` in context names.

**Comment by idcmp** [comment]
@loganmzz Can you confirm by "no issue", you mean you're generating the autocompletion and using it via tab successfully?

**Comment by loganmzz** [comment]
Yes all my context names contains a `@` and autocomplete nicely.
